dhis2/src/main/java/org/motechproject/dhis2/rest/service/impl/DhisWebServiceImpl.java

```diff
@@ -511,6 +548,57 @@ private DhisDataValueStatusResponse importDataValues(Settings settings, String u
             String msg = String.format("Error parsing response from uri: %s, exception: %s", uri, e.toString());
             statusMessageService.warn(msg, MODULE_NAME);
             throw new DhisWebException(msg, e);
+        } finally {
+            closeResponse(response);
+        }
+    }
+
+    private void closeResponse(CloseableHttpResponse response) {
+        try {
+            if (response != null) {
+                EntityUtils.consume(response.getEntity());
+                response.close();
+            }
+        } catch (IOException e) {
+            LOGGER.error("Error trying to close response: " + e.getMessage());
```

Comment:
we want to print to ex with  the stracktrace, so do LOGGER.error("Error trying to close response", e) instead.